File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void build(int x);


int main(void)
{
    int x, y, input; //x=Outer counter/ y=inner counter

    do
    {
        input = get_int("How tall do you want this pyramid to be. Max 38\n");
        if (input < 0)
        {
            printf("no negative values\n");
        }
        if (input == 0)
        {
            printf("no Zero values\n");
        }
        if (input > 8)
        {
            printf("Maximum size is 8\n");
        }
    }
    while (input < 1 || input > 8);

    for (x = 1; x <= input; x++)
    {

        for (y = input - 1; y >= x; y--)//offsets 1st half of the pyramid
        {
            printf(" ");
        }
        build(x);         //Builds 1st half of the pyramid
        printf("  ");       //empty space between pyramid
        build(x);         //Builds 2nd half of the pyramid
        printf("\n");       //New line
    }
}
void build(int x)
{
    int i;
    for (i = 0; i < x; i++)
    {
        printf("#");
    }
}

File: sentimental-mario-more/mario.py
import cs50
while (True):
    input = cs50.get_int("Input an integer: ")
    if input>0 and input<9:
        break

for x in range(input):
    for y in range (input-1, x, -1):
        print(" ", end = '')
    for y in range (x+1):
        print("#", end = '')
    print("  ", end = '')
    for y in range (x+1):
        print("#", end = '')
    print()
print()